src/Service.Tests/SqlTests/RestApiTests/Patch/PatchApiTestBase.cs

```diff
@@ -188,17 +180,16 @@ public virtual async Task PatchOne_Insert_NonAutoGenPK_Test()
                 ""Scientific Name"": ""Quercus"",
                 ""United State's Region"": ""South West""
             }";
-            expectedLocationHeader = $"treeId/4";
 
             await SetupAndRunRestApiTest(
-                    primaryKeyRoute: expectedLocationHeader,
+                    primaryKeyRoute: $"treeId/4",
                     queryString: null,
                     entityNameOrPath: _integrationMappingEntity,
                     sqlQuery: GetQuery("PatchOne_Insert_Mapping_Test"),
                     operationType: EntityActionOperation.UpsertIncremental,
                     requestBody: requestBody,
                     expectedStatusCode: HttpStatusCode.Created,
-                    expectedLocationHeader: expectedLocationHeader
+                    expectedLocationHeader: string.Empty
```

Comment:
Why dont we expect a location header now for Insert Patch requests? Why did we expect previously?

Reply:
I've explained this in detail in an earlier [thread](https://github.com/Azure/data-api-builder/pull/1731#discussion_r1339595187)